app/navigation/OtherOfficialsNavigator.scala

```diff
@@ -29,7 +28,7 @@ import play.api.mvc.Call
 
 import javax.inject.Inject
 
-class OtherOfficialsNavigator @Inject() (implicit frontendAppConfig: FrontendAppConfig) extends BaseNavigator {
```

Comment:
This can just be class `OtherOfficialsNavigator extends BaseNavigator {`. Please remove import also. Thanks